samples/openapi3/client/petstore/python-pydantic-v1/tests/test_api_client.py

```diff
@@ -9,8 +9,6 @@
 $ pytest
 """
 
-import os
-import time
```

Comment:
Can you revert this change? It has nothing to do with your original PR.

Reply:
yep yep